app/brave_settings_strings.grdp

```diff
@@ -1571,9 +1571,6 @@ Leo AI. This action can't be undone.
   <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_VISION_SUPPORT" desc="An input title for model vision support.">
     Vision Support
   </message>
-  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_VISION_SUPPORT_TOOLTIP_INFO" desc="A brief description for model vision support for adding a custom model.">
```

Comment:
You can't remove this string, it is for BYOM model setting to indicate vision support